### PR TITLE
Update building and running workflows

### PR DESCRIPTION
**Pull request by StefanJum**
* Add default minimal configs that can be used as staring point
* Add running scripts
* Change `UK_ROOT` and `UK_LIBS` to point to `$(PWD)/.unikraft/...` [TODO]: Add README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,319 @@
...
+In order to set up, configure, build and run SQLite on Unikraft, the following packages are required:
+
+* `build-essential` / `base-devel` / `@development-tools` (the meta-package that includes `make`, `gcc` and other development-related packages)
+* `sudo`
+* `flex`
+* `bison`
+* `git`
+* `wget`
+* `uuid-runtime`
+* `qemu-system-x86`
```
Comment (razvand) [inline README.md:17]:
```suggestion
* `qemu-system-x86`
* `qemu-system-arm`
```

File: README.md [hunk 2]
```diff
@@ -1,50 +1,319 @@
...
+  gcc-aarch64-linux-gnu \
+  libncurses-dev \
+  libyaml-dev \
+  flex \
+  bison \
+  git \
+  wget \
+  uuid-runtime \
+  qemu-kvm \
+  qemu-system-x86 \
```
Comment (razvand) [inline README.md:40]:
```suggestion
  qemu-system-x86 \
  qemu-system-arm \
```

File: README.md [hunk 3]
```diff
@@ -1,50 +1,319 @@
...
+* `qemu-kvm`
+* `sgabios`
+* `gcc-aarch64-linux-gnu`
+
+GCC >= 8 is required to build SQLite on Unikraft.
+
+On Ubuntu/Debian or other `apt`-based distributions, run the following command to install the requirements:
+
+```console
+$ apt-get install -y --no-install-recommends \
```
Comment (razvand) [inline README.md]:
```suggestion
$ sudo apt install -y --no-install-recommends \
```

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,10 @@
+UK_ROOT ?= $(PWD)/.unikraft/unikraft
+UK_LIBS ?= $(PWD)/.unikraft/libs
+LIBS := $(UK_LIBS)/musl:$(UK_LIBS)/sqlite
+
+all:
+	@$(MAKE) -C $(UK_ROOT) A=$(PWD) L=$(LIBS)
+
+$(MAKECMDGOALS):
+	@$(MAKE) -C $(UK_ROOT) A=$(PWD) L=$(LIBS) $(MAKECMDGOALS)
+
```
Comment (razvand) [inline Makefile]:
```suggestion
```

File: Makefile.uk [hunk 5]
```diff
@@ -0,0 +1,2 @@
+$(eval $(call addlib,appsqlite))
+
```
Comment (razvand) [inline Makefile.uk]:
```suggestion
```

**Review by razvand — CHANGES_REQUESTED** [review]
Add `.gitignore` file to ignore output files.

**Comment by StefanJum** [comment]
Done @razvand

File: run-qemu-aarch64.sh [hunk 6]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (razvand) [inline run-qemu-aarch64.sh]:
```suggestion
#!/bin/sh
```

File: run-qemu-x86_64.sh [hunk 7]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
Comment (razvand) [inline run-qemu-x86_64.sh]:
```suggestion
#!/bin/sh
```

File: run-qemu-x86_64.sh [hunk 8]
```diff
@@ -0,0 +1,6 @@
...
+
+sudo /usr/bin/qemu-system-x86_64 \
```
Comment (razvand) [inline run-qemu-x86_64.sh]:
```suggestion
/usr/bin/qemu-system-x86_64 \
```

File: run-qemu-aarch64.sh [hunk 9]
```diff
@@ -0,0 +1,7 @@
...
+
+sudo /usr/bin/qemu-system-aarch64 \
```
Comment (razvand) [inline run-qemu-aarch64.sh]:
```suggestion
/usr/bin/qemu-system-aarch64 \
```

**Review by razvand — CHANGES_REQUESTED** [review]
Thanks, @StefanJum. See my comments.

**Review by razvand — APPROVED** [review]
Reviewed-by: Razvan Deaconescu <razvand@unikraft.io>
Approved-by: Razvan Deaconescu <razvand@unikraft.io>

**Pull request closed without merging**